pkg/scalers/redis_scaler.go

```diff
@@ -0,0 +1,143 @@
+package scalers
+
+import (
+	"context"
+	"fmt"
+	"strconv"
+
+	log "github.com/Sirupsen/logrus"
+	"github.com/go-redis/redis"
+	v2beta1 "k8s.io/api/autoscaling/v2beta1"
+	"k8s.io/apimachinery/pkg/api/resource"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/labels"
+	"k8s.io/metrics/pkg/apis/external_metrics"
+)
+
+const (
+	listLengthMetricName    = "RedisLengthOfQueue"
+	defaultTargetListLength = 5
+	defaultRedisAddress     = "redis"
```

Comment:
Why is this a default redis address? Its not a network addressable value..?

Reply:
@yaron2 added a default address of redis-master.default.svc.cluster.local:6379